TelenorNBIoT.cpp

```diff
@@ -366,52 +366,47 @@ bool TelenorNBIoT::sendString(IPAddress remoteIP, const uint16_t port, String st
     return sendBytes(remoteIP, port, str.c_str(), str.length());
 }
 
-bool TelenorNBIoT::receiveFrom(char *ip, uint16_t *port, char *outbuf, uint16_t *length, uint16_t *remain)
+size_t TelenorNBIoT::receiveBytes(char *buffer, uint16_t bufferLength)
```

Comment:
`buffer` shadows `this.buffer`.  Rename it back to `outbuf`.

Reply:
Good catch 🕵️